packages/design-system-web/src/components/geov-entity/geov-entity.tsx

```diff
@@ -133,6 +138,11 @@ export class GeovEntity {
             uriReplace={this.uriReplace}
           ></geov-entity-properties>
 
+          {/* Download RDF button */}
+          <div class="section columns-1 ion-text-center">
+            <geov-entity-download-rdf entityId={this.entityId} projectId={this.projectId} color="primary" fill="outline" buttonLabel="Download RDF" buttonIcon="download-outline" />
```

Comment:
The button has no styles for darkmode. But the entity-component has. Therefor it will be almost invisible on darkmode:
![darkmode-screenshot](https://github.com/geovistory/design-system/assets/33655122/ec0f9c0b-8203-4927-87aa-fbc0f7b2c65b)


Please add styles for darkmode-preference. @joschne How would you do that? should the primary color be another one in darkmode?

Reply:
As Jonas said, we don't resolve in this PR.